lib/Devel/KYTProf/Profiler/Furl/HTTP.pm

```diff
@@ -8,13 +8,32 @@ sub apply {
         'Furl::HTTP',
         'request',
         sub {
-            my($orig, $self, %args) = @_;
+            my ($orig, $self, %args) = @_;
+
+            my $method = $args{method} || 'GET';
+            my $url = do {
```

Comment:
[nit]
Rather than using "do", I think it is better as below for making the nesting shallower. How about it?